charts/consul/templates/_helpers.tpl

```diff
@@ -48,7 +48,7 @@ as well as the global.name setting.
 {{- define "consul.serverTLSAltNames" -}}
 {{- $name := include "consul.fullname" . -}}
 {{- $ns := .Release.Namespace -}}
-{{ printf "localhost,%s-server,*.%s-server,*.%s-server.%s,*.%s-server.%s.svc,*.server.%s.%s" $name $name $name $ns $name $ns (.Values.global.datacenter ) (.Values.global.domain) }}{{ include "consul.serverAdditionalDNSSANs" . }}
+{{ printf "localhost,%s-server,*.%s-server,*.%s-server.%s,*.%s-server.%s.svc,%s-server.%s.svc,*.server.%s.%s" $name $name $name $ns $name $ns $name $ns (.Values.global.datacenter ) (.Values.global.domain) }}{{ include "consul.serverAdditionalDNSSANs" . }}
```

Comment:
Modifying SANs in cert so client pod can talk to server load balancer.

Reply:
Those are the 2nd and 4th entries above.

Reply:
Oh good point! I didn't notice the 2nd entry. The 4th one though is `*.%s-server.%s` and I'm wondering about `%s-server.%s` (i.e. without a wildcard).

Reply:
Just pushed it up.